Servers/UI/OJS.Servers.Ui/ClientApp/src/styles/_responsive.scss

```diff
@@ -0,0 +1,19 @@
+@use 'src/styles/variables';
+
+%r-heading-primary {
+  @media only screen and (min-width: variables.$r-small-max) and (max-width: variables.$r-large-max) {
+    font-size: 1.8vw;
```

Comment:
I am not a fan of `vw` and `vh`. They might look OK in the happy case, i.e. fullscreen, but otherwise they break and need a lot of configuration. Some examples:
1. Half width for browser, other half for VS code
2. Half height for browser, other half for terminal
3. Combination of the above

Use pixels

Reply:
Dont even know why i've used vw for font but oh well 🤦‍♀️